doc/theming.rst

```diff
@@ -20,6 +20,10 @@ the theme's look and feel.
 Themes are meant to be project-unaware, so they can be used for different
 projects without change.
 
+.. note::
```

Comment:
This entire document should be moved to the development directory, and the information on injecting Javascript files included there.

Reply:
Do you want me to do that as a part of this PR?

Reply:
I'll move it and see what you think